eng/pipelines/pr.yml

```diff
@@ -33,6 +33,7 @@ variables:
   NUGET_EXPERIMENTAL_CHAIN_BUILD_RETRY_POLICY: 3,1000
   SkipSigning: true
   CI: true
+  Codeql.Enabled: false
```

Comment:
So, we only scan in the "official" pipeline?

Reply:
We can scan here, but these jobs are all for running tests.  I think I'll add a new job that just builds and runs CodeQL or all SDL tooling.  We just don't need to run it for the current jobs because their intent is to run tests